study-builder/studies/osteo/emails/exit_request.html

```diff
@@ -75,7 +74,7 @@
   <tr style="width: 540px;">
     <td align="left" style="margin: 0; padding: 50px 0 0 0; font-family: 'Source Sans Pro', sans-serif;">
       <a href="-ddp.baseWebUrl-" target="_blank" style="cursor: pointer; text-decoration: none;">
-        <img src="https://storage.googleapis.com/cmi-study-dev-assets/osteo/project-logo.png"
+        <img src="https://storage.googleapis.com/${assetsBucketName}/osteo/project-logo.png"
```

Comment:
We can do a lot more than just substituting simple values. Using Velocity, we can do templating of whole html snippets if we wanted to, or do logic like `#if` and `#foreach`.

Reply:
Oh yes please!  Whole snippets like header, footer!

Reply:
Since I just manually updated dozens of copies of the footer 😢

Reply:
We could also simplify some of the first reminder, 2nd reminder, 3rd reminder copy-pasta too. Like `"${textSinceStart}, we sent you a reminder about...`"